### DISCUSSION TITLE
Extend "open" option to other than "url"

### DISCUSSION THREAD

**Discussion by notresponding2u** [ayushnix/tessen#44] [Ideas]
Hi, 

I love the script. And of course, I am not using it mainly for passwords, which is probably not a surprise. 

I use the script mainly for my job, because there are hundreds of passwords. I also find it very useful as a bookmark tool, which is not the main point of pass or tessen. But I can't deny it is very useful to handle almost everything else that I need in reach.

Some of the files I have contain urls. However, if there is no "url" key the "open" option doesn't show up in the third menu. 

My humble suggestion is to add a keyword, which in case found as a substring (like `something_url`) in the key, will set the `key_arr=("open" "copy")` for the third menu.

I would appreciate if you enable such feature, so I can misuse the tool better.

Sincerely.

**Comment by ayushnix** [comment]
> My humble suggestion is to add a keyword, which in case found as a substring (like something_url) in the key, will set the key_arr=("open" "copy") for the third menu.

You can choose a custom keyword for the URL key in the config file. The value of `urlkey` in the tessen config file can be set to anything you want and when that value is found in a file, tessen will use the `open` and `copy` options.
